docs/source/guides/getting_started.rst

```diff
@@ -226,7 +226,7 @@ HelloWorldDto
     }
   }
 
-This class is much less exciting than the `HelloWorldResource`_. The few things to note here are the
+This class defines the JSON response from the service. The few things to note here are the
```

Comment:
Do the examples need to be updated for swagger 2?

Reply:
I think only the packages changed but the class names stayed the same, so if the examples don't include the import statement, it should be fine.